src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-console */
+import { FC, useEffect, useState } from 'react';
+
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+import { fetchPeople } from '../../api';
+import { Person } from '../../types';
+
+const getParents = (
+  { motherName, fatherName }: Person,
+  array: Person[],
+) => ({
+  mother: array.find(({ name }) => name === motherName),
+  father: array.find(({ name }) => name === fatherName),
```

Comment:
what if parent is not found?

Reply:
null (I think) is passed by default and I make a ternary check in PeopleTable for it